csxtools/utils.py

```diff
@@ -224,10 +224,10 @@ def get_fastccd_timestamps(header):
         list of arrays of the timestamps
 
     """
-    hover = {'fccd_image_lightfield': AreaDetectorHDF5TimestampHandler}
-    img = [i for i in get_events(header, ['fccd_image_lightfield'],
+    hover = {tag: AreaDetectorHDF5TimestampHandler}
+    img = [i for i in get_events(header, [tag],
                                  handler_overrides=hover)]
-    timestamps = img[0]['data']['fccd_image_lightfield']
+    timestamps = [i['data'][tag] for i in img if tag in i['data'].keys()]
```

Comment:
I see what you say, but this does not place `np.nan` for fccd timestamps that don't exist but rather for events that don't have a `tag`. Your suggestion would make the output list be of length of all events, not just events that have a `tag`. Perhaps both behaviors would be best to add with a `fill` kwarg